src/main/java/taxi/controller/users/LoginController.java

```diff
@@ -0,0 +1,41 @@
+package taxi.controller.users;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+
+public class LoginController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final AuthenticationService authenticationService = (AuthenticationService) injector
+            .getInstance(AuthenticationService.class);
+
+    @Override
+    public void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/users/login.jsp").forward(req, resp);
+    }
+
+    @Override
+    public void doPost(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String userName = req.getParameter("login");
+        String password = req.getParameter("password");
+        try {
+            Driver driver = authenticationService.login(userName, password);
+            HttpSession session = req.getSession();
+            session.setAttribute("user_id", driver.getId());

```

Comment:
you use `Driver` model
```suggestion
            session.setAttribute("driver_id", driver.getId());

```